src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1302,7 +1302,14 @@ private void PlaceIfPossible(string organelleType, int q, int r, int rotation)
     private bool IsValidPlacement(OrganelleTemplate organelle)
     {
         bool notPlacingCytoplasm = organelle.Definition.InternalName != "cytoplasm";
-        return editedMicrobeOrganelles.CanPlaceAndIsTouching(organelle, notPlacingCytoplasm);
+
+        // Allow deleting the last cytoplasm if an organelle is about to be placed
+        bool allowReplacingLast = true;
+
+        return editedMicrobeOrganelles.CanPlaceAndIsTouching(
+            organelle,
+            notPlacingCytoplasm,
+            allowReplacingLast);
```

Comment:
I think this should be just
`return editedMicrobeOrganelles.CanPlaceAndIsTouching(organelle, notPlacingCytoplasm, notPlacingCytoplasm);`
so the replace of last cytoplasm is allowed when placing something else than cytoplasm.

Reply:
Switched to that in a766ffb